src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <main>
+      <div class="card" data-qa="card">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <div class="card__description">
+          <div>
+            <h2 class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__product-code">
+              Product code: 195434
+            </p>
+          </div>
+
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+            </div>
+            <div>Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div>
+              Price:
+            </div>
+            <div class="card__price-current">
+              $2,199
+            </div>
+          </div>
+          <a
+          class="card__buy-button"
+          data-qa="hover"
+          href="#buy"
+          >
```

Comment:
```suggestion
          <a
            class="card__buy-button"
            data-qa="hover"
            href="#buy"
          >
```
don't forget about indentions